src/components/PostVolunteer/index.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Col } from "react-bootstrap";
+import "./index.css";
+
+export default function PostVolunteer(postInfo) {
+  const { t } = useTranslation();
+  if (postInfo) {
+    return (
+      <Col className="singlePost" lg={3} sm={12} md={6}>
+        <p className="cardTitle text-center">
+          {postInfo.postInfo ? postInfo.postInfo.title : ""}
```

Comment:
You can use conditional rendering here instead of ternary operator i.e   `{postInfo.postInfo && postInfo.postInfo.title}`